quaddtype/numpy_quaddtype/src/casts.cpp

```diff
@@ -401,7 +401,7 @@ to_quad<long double>(long double x, QuadBackendType backend)
 }
 
 template <typename T>
-static NPY_CASTING
+static int
```

Comment:
Hmm... I think this is OK? I didn't see any new warnings, but there are thousands of build warnings when using sleef and I might have missed it.